python/pyspark/sql/connect/window.py

```diff
@@ -218,27 +218,201 @@ class Window:
 
     @staticmethod
     def partitionBy(*cols: Union["ColumnOrName", List["ColumnOrName"]]) -> "WindowSpec":
+        """
```

Comment:
You won't need this. the doc is copied below `partitionBy.__doc__ = PySparkWindow.partitionBy.__doc__` and the doctest runs.